index.bs

```diff
@@ -320,8 +322,15 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent may add an additional field to the TXT record:
+
+: pw
+:: An alphanumeric, unguessable token consisting of a [=Base64=] encoded 128-bit
+    random value (24 characters in length).
 
+Note: The requirements for `pw` are the same as for an [=ICE password=].
```

Comment:
Not sure what this buys us.  It doesn't seem worth having a dependency on ICE.

Reply:
Agreed. Removed reference to ICE and just recommend a minimum entropy.  We can discuss the exact value at the next F2F.